artemis-web/src/main/java/org/apache/activemq/artemis/component/WebServerComponent.java

```diff
@@ -72,6 +74,11 @@ public void configure(ComponentDTO config, String artemisInstance, String artemi
          SslContextFactory.Server sslFactory = new SslContextFactory.Server();
          sslFactory.setKeyStorePath(webServerConfig.keyStorePath == null ? artemisInstance + "/etc/keystore.jks" : webServerConfig.keyStorePath);
          sslFactory.setKeyStorePassword(webServerConfig.getKeyStorePassword() == null ? "password" : webServerConfig.getKeyStorePassword());
+         sslFactory.setIncludeProtocols("TLSv1.2");
```

Comment:
I'm concerned about changing the default here for all cases. This should probably be made configurable and then the test can configure it as necessary.

Reply:
I added some parameters to the WebServerDTO so the test can configure them.